src/runtimes/wasi_cli/runtime.rs

```diff
@@ -119,11 +119,19 @@ impl<'a> Runtime<'a> {
                     )
                 }
                 match serde_json::from_slice::<PolicyValidationResponse>(stdout.as_bytes()) {
-                    Ok(pvr) => AdmissionResponse::from_policy_validation_response(
-                        request.uid().to_string(),
-                        request.0.get("object"),
-                        &pvr,
-                    )
+                    Ok(pvr) => {
+                        let req_json_value = serde_json::to_value(request).unwrap();
```

Comment:
Can you avoid doing the `unwrap` here? We should return a rejection because of an internal error

Reply:
this error can never happen since the request is serializable

Reply:
> This conversion can fail if T’s implementation of Serialize decides to fail, or if T contains a map with non-string keys.

We can rule out the 2nd case, but we cannot assume the serialization of the request will always be successful. I think there are corner cases like memory allocation errors and similar. Am I too paranoid?

Reply:
agreed on `except` in Slack